pubspec.yaml

```diff
@@ -9,4 +9,4 @@ dependencies:
   browser: '>=0.9.0 <0.11.0'
   quiver: '>=0.18.2 <0.22.0'
 dev_dependencies:
-  unittest: '>=0.11.0'
+  unittest: '< 0.12.0'
```

Comment:
this should probably keep the low end of the acceptable range too just in case even older versions don't work (or if older version don't actually work, can pin to a strict version):
```
unittest: '>=0.11.0 <0.12.0'
```